isis/src/base/apps/isisimport/BaseProcessFunctor.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BASE_POST_PROCESS_FUNCTOR_H
```

Comment:
Thinking we should name this base class better. The current one doesn't give me much to know what it does.

Reply:
Renamed to `isisimportBaseFunctor`